packages/next-on-pages/templates/_worker.js/index.ts

```diff
@@ -12,25 +15,29 @@ declare const __ENV_ALS_PROMISE__: Promise<null | AsyncLocalStorage<unknown>>;
 
 export default {
 	async fetch(request, env, ctx) {
+		patchFetchToAllowBundledAssets();
```

Comment:
Does this need to happen in the fetch handler? Couldn't this happen in the global scope?

Reply:
yes, I thought doing it here felt cleaner, I've also added a flag in the function so that we don't do multiple patches, so even though this can run multiple times I don't think it really introduces any overhead

but anyways I can just as well move it in the global scope if you prefer, I don't have a strong opinion on it 🙂 

(but I guess I'd leave the flag there anyways just in case? or would it become completely unnecessary in the global scope?)

Reply:
I'd move it. The work we're doing in this isn't dependent on the request, and it is supposed to be mutating the global from the onset. So to me, it makes sense to live in the global space.